build/Common.props

```diff
@@ -46,7 +46,7 @@
     <SystemDiagnosticSourcePkgVer>6.0.0</SystemDiagnosticSourcePkgVer>
     <SystemReflectionEmitLightweightPkgVer>4.7.0</SystemReflectionEmitLightweightPkgVer>
     <SystemTextJsonPkgVer>4.7.0</SystemTextJsonPkgVer>
-    <SystemThreadingTasksExtensionsPkgVer>4.5.3</SystemThreadingTasksExtensionsPkgVer>
+    <SystemThreadingTasksExtensionsPkgVer>4.5.4</SystemThreadingTasksExtensionsPkgVer>
```

Comment:
This is required, otherwise it will cause a build failure in the Jaeger Exporter:

OpenTelemetry.Exporter.Jaeger.csproj : error NU1605: Detected package downgrade: System.Threading.Tasks.Extensions from 4.5.4 to 4.5.3. Reference the package directly from the project to select a different version.